tests/datm_conf/cpld_datm_mom6_cice5_cfsr.IN

```diff
@@ -0,0 +1,24 @@
+mkdir INPUT DATM_INPUT RESTART history MOM6_OUTPUT MOM6_RESTART restart
```

Comment:
I don't believe we are still using _MOM6_RESTART_ or _restart_; both MOM6 and CICE5 now write their restarts to the RESTART directory.

Reply:
These two directories will be removed.